charmcraft.yaml

```diff
@@ -24,6 +24,14 @@ provides:
   cos-agent:
     interface: cos_agent
 
+# These bindings can be used to explicitly request interfaces in all the OpenStack
+# network spaces. Needed when APIs are not exposed in the same network as the one bound
+# to the credentials endpoint
+extra-bindings:
+  admin-interface:
+  public-interface:
+  internal-interface:
```

Comment:
Is there any strong reason not to use the same naming convention with the previous charm and other OpenStack charms?
- https://git.launchpad.net/charm-prometheus-openstack-exporter/commit/?id=99692c082eadbf5195bb683e92a73428603efbc6
- https://opendev.org/openstack/charm-keystone/src/branch/master/metadata.yaml#L15-L18

Reply:
not a strong one; I am proposing the `-interface` suffix to be a bit more clear that there's nothing special about these bindings, it's just a trick to ensure the unit is created with extra interfaces

Reply:
To me, it's an unnecessary diff. This charm resides in the Charmed OpenStack ecosystem so I don't see a necessity to diverge from the standard naming convention.

Reply:
Agreed, let's stick to convention to avoid confusion.  We can add info to the charmhub docs about the purpose of the extra bindings if needed.

Reply:
sure